pages/main-page.ts

```diff
@@ -0,0 +1,22 @@
+export class MainPage {
+  readonly page: Page;
+  constructor(page: Page) {
+    this.page = page;
+    this.setStorage = page.getByRole('link', { name: 'Storage' });
```

Comment:
always add the type of UI control after the name, ie: setStorageLink

Reply:
updated